e2e/.gitignore

```diff
@@ -0,0 +1,2 @@
+# ignore directory for artifacts produced by tests
+test-results/
```

Comment:
I refer to add it to the root .ignore file

Reply:
done